packages/react-ui-components/src/stories/Components/TabNavigation.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { ComponentStory } from '@storybook/react';
+import React, { useState } from 'react';
+import { KvTabNavigation } from '../../components';
+
+// Required to have the correct TagName in the code sample
+KvTabNavigation.displayName = 'KvTabNavigation';
+
+export default {
+	title: 'Components/Tab Navigation',
+	component: 'kv-tab-navigation',
+	argTypes: {
+		selectedTabKey: {
+			control: 'text'
+		}
+	},
+	parameters: {
+		notes: require('@ui-notes/tab-navigation/readme.md')
+	}
+};
+
+const TabNavigationTemplate: ComponentStory<typeof KvTabNavigation> = args => {
+	const [selectedTab, setTab] = useState(args.selectedTabKey);
+
+	const handleTabChange = (event: CustomEvent<string>) => {
+		setTab(event.detail);
+	};
+
+	return <KvTabNavigation tabs={args.tabs} selectedTabKey={selectedTab} onTabChange={handleTabChange} notifications={args.notifications}></KvTabNavigation>;
```

Comment:
should we split this line for an easy reading?

Reply:
linter will fail